KITTI_visual_odometry.ipynb

```diff
@@ -2362,9 +2362,9 @@
     "        self.poses_dir = '../dataset/poses/{}.txt'.format(sequence)\n",
     "        poses = pd.read_csv(self.poses_dir, delimiter=' ', header=None)\n",
     "        \n",
-    "        # Get names of files to iterate through\n",
-    "        self.left_image_files = os.listdir(self.seq_dir + 'image_0')\n",
-    "        self.right_image_files = os.listdir(self.seq_dir + 'image_1')\n",
+    "        # Get sorted names of files to iterate through\n",
+    "        self.left_image_files = sorted(os.listdir(self.seq_dir + 'image_0'))\n",
+    "        self.right_image_files = sorted(os.listdir(self.seq_dir + 'image_1'))\n",
```

Comment:
Good call. Bringing this one step further, to avoid any ambiguity about how strings are sorted across different systems, let's add a key function converting the file names to ints for sorting.
```suggestion
    "        self.left_image_files = sorted(os.listdir(self.seq_dir + 'image_0'), key=lambda x: int(x.split(".")[0]))\n",
    "        self.right_image_files = sorted(os.listdir(self.seq_dir + 'image_1'), key=lambda x: int(x.split(".")[0]))\n",
```